src/main/java/org/zendesk/client/v2/model/hc/ArticleAttachments.java

```diff
@@ -9,7 +9,7 @@
     /**
      *  Automatically assigned when the article attachment is created
      */
-    private int id;
+    private Integer id;
```

Comment:
https://developer.zendesk.com/rest_api/docs/help_center/article_attachments
All fields marked as integer in the doc should be Long in Java
See PR#170